assignment/guestbook/models.py

```diff
@@ -0,0 +1,33 @@
+from django.db import models
+
+
+class Post(models.Model):
+    """
+    Model representing a single post in the guestbook.
+    The post consists of
+        - the name of the person
+        - a comment
+        - a contact email
+        - the timestamp of when the post was created
+        - the timestamp of when the post was hidden (if at all)
+    """
+    name = models.CharField(max_length=254)
+    text = models.CharField(max_length=1000)
+    email = models.EmailField(blank=True)
+    created_at = models.DateTimeField(auto_now=True)
```

Comment:
`auto_now` will set the field to the current time every time the object is saved. This would work well for `modified_at`.

What you want in this case is `auto_now_add`, which will save the time when the `Post` is first created.